### ISSUE TITLE
Maintable beta version always freeze when attaching new file

### ISSUE THREAD

**Issue by psychelzh** [JabRef/jabref#3858]
Just like this.

![image](https://user-images.githubusercontent.com/13306020/37550785-18879a52-29ce-11e8-9c25-1432dad07972.png)

```
JabRef 4.2-dev--snapshot--2018-03-16--maintable-beta--2c2a302a5
Windows 10 10.0 amd64 
Java 1.8.0_161
```

**Comment by Siedlerchr** [comment]
Will hopefully soon be fixed with #3005

**Comment by Siedlerchr** [comment]
And as a workaround, if you directly go to the General tab -> Add file (the + symbol)  it works fine

**Comment by psychelzh** [comment]
Thanks. I will use this workaround for now.

**Comment by Siedlerchr** [comment]
Fixed with #3005
